### PR TITLE
[WHL] Split memory SPD CFGDATA into sub-pages for display

### PR DESCRIPTION
**Pull request by mauricema**
Whiskeylake platform contains memory SPD data in memory configurations.
Since it has 4 SPD tables, loading is very slow in ConfigEditor. The SPD
table can be broken down into individual page for display to enhance the
loading performance in ConfigEditor.

Signed-off-by: Maurice Ma <maurice.ma@intel.com>

### CODE REVIEW

File: Platform/CoffeelakeBoardPkg/CfgData/CfgData_Memory.dsc [hunk 1]
```diff
@@ -68,7 +76,7 @@
   # !BSF NAME:{Dqs Pins Interleaved Setting} TYPE:{Combo} OPTION:{$EN_DIS}
   # !BSF HELP:{Indicates DqPinsInterleaved setting: board-dependent}
   gCfgData.DqPinsInterleaved                |      * | 0x01 | 0x1
-  
+
   # !BSF NAME:{Tseg Size} TYPE:{Combo}
```
Comment (myguitar) [inline Platform/CoffeelakeBoardPkg/CfgData/CfgData_Memory.dsc]: This Tseg has invalid value(0x800000) and I think this should be removed and controlled in code directly because
there is description "0x400000 for Release build and 0x1000000 for Debug build" in FspmUpd.h.
Let me do it in separate commit.

Comment (mauricema) [reply Platform/CoffeelakeBoardPkg/CfgData/CfgData_Memory.dsc]: Thanks!  Sure, let us fix that separately.
